fs/read.js

```diff
@@ -1,16 +1,35 @@
+var http = require('http')
 var fs = require('fs');
 
 // Use fs module for all operatins below.
 
 
 // 1. Write script to read file theory.md and console the output buffer.
+fs.readFile('./theory.md', (err,data)=>{
+  if(err) console.log(err) 
+  if(data) console.log(data)
+})
 
 // 2. Write script to read file theory.md and convert resulted buffer using toString
   // method and console the result.
+  fs.readFile('./theory.md', (err,data) => {
+    if(err) console.log(err) 
+    if(data) console.log(data.toString())
+  })
 
 // 3. Write script to read file Synchronously and console the output.
+var fileSync = fs.readFileSync('./theory.md')
+console.log(fileSync)
 
 // 4. Write script to create a file 'write.js' and write content of read.js in there.
+var readFile = fs.readFileSync('./read.js', 'utf-8', (err, data) => {
```

Comment:
Sync methods do not return a callback function. It returns the result immediately which can be stored into a variable.
It should be
```js
var readFile = fs.readFileSync('./read.js', 'utf-8');
```